op-challenger/game/keccak/challenger.go

```diff
@@ -80,7 +79,7 @@ func (c *PreimageChallenger) Challenge(ctx context.Context, blockHash common.Has
 	wg.Wait()
 	c.log.Debug("Created preimage challenge transactions", "count", len(txs))
 	if len(txs) > 0 {
-		_, err := c.sender.SendAndWait("challenge preimages", txs...)
+		err := c.sender.SendAndWaitSimple("challenge preimages", txs...)
 		if err != nil {
```

Comment:
Nit, should be able to inline this err declaration and check.
```suggestion
		if err := c.sender.SendAndWaitSimple("challenge preimages", txs...); err != nil {
```

Reply:
Will follow up with another pr not to block